dexcom/fetch/runner.go

```diff
@@ -26,10 +26,10 @@ import (
 )
 
 const (
-	AvailableAfterDurationMaximum = 75 * time.Minute
-	AvailableAfterDurationMinimum = 45 * time.Minute
+	AvailableAfterDurationMaximum = 135 * time.Minute
+	AvailableAfterDurationMinimum = 105 * time.Minute
 	DataSetSize                   = 2000
-	TaskDurationMaximum           = 5 * time.Minute
+	TaskDurationMaximum           = 10 * time.Minute
```

Comment:
Doubled task duration to account for a couple of long running errors observed in logs.

Reply:
> Doubled task duration to account for a couple of long running errors observed in logs.

I think that'd be useful to capture in the commit message, to help future us. Same with your other comment in this PR. Just $0.02 though.